### PR TITLE
Signing authority

### PR DESCRIPTION
**Pull request by RuvendeGroote**

### CODE REVIEW

File: src/store/api.ts [hunk 1]
```diff
@@ -242,11 +246,21 @@ export const getDepositorInfoAccountData = multiAsync(
 );
 
 export const createDepositor = multiAsync(async (connection: Connection, wallet: Wallet) => {
```
Comment (MaximPiessen) [inline src/store/api.ts:231]: This fails when I try to run the app and deposit funds

Comment (MaximPiessen) [reply src/store/api.ts:231]: In the tests, you do: 
```
const [signingAuthorityPda, _] = await anchor.web3.PublicKey.findProgramAddress(
      [Buffer.from(program.programId.toString()).subarray(0, 32)],
      program.programId
    );
```
In the app you do: 
```
const findSigningAuthorityPDA = multiAsync(async () => {
	const seeds: PdaSeeds = [config.clusterConfig.programId.toBuffer()];
	return findPDA(seeds);
});
```
=> are those the same?

Comment (RuvendeGroote) [reply src/store/api.ts:231]: no, we need the subarray version. I'll fix

File: src/store/api.ts [hunk 2]
```diff
@@ -472,20 +475,43 @@ const getLPTokenPrice = multiAsync(async (connection: Connection, wallet: Wallet
...
+	return Number(userLPTokenAccount.account.data.parsed.info.tokenAmount.amount);
+});
+
+export const getLPTokenUSDCBalance = multiAsync(async (connection: Connection, wallet: Wallet) => {
+	const _lpTokenPrice = getLPTokenPrice(connection, wallet);
+	const _userLPTokenAmount = getUserLPTokenAmount(connection, wallet);
+
+	const [lpTokenPrice, userLPTokenAmount] = await Promise.all([_lpTokenPrice, _userLPTokenAmount]);
+
+	console.log("token", userLPTokenAmount);
```
Comment (BouweCeunen) [inline src/store/api.ts]: just a small thing, maybe remove this?

**Review by BouweCeunen — DISMISSED** [review]

**Review by BouweCeunen — APPROVED** [review]

**Review by MaximPiessen — APPROVED** [review]

**Pull request merged**